doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/03_Full_Page_Cache.md

```diff
@@ -53,6 +53,10 @@ In this case you can use obtain the pull page cache service from the container a
 ```php
 <?php
 $this->get(\Pimcore\Bundle\CoreBundle\EventListener\Frontend\FullPageCacheListener::class)->disable("Your disable reason");
+
+or better with DI 
```

Comment:
I would replace `$this->get` example with DI as this anyway wont work with v11. see https://github.com/pimcore/pimcore/pull/12048

Reply:
yes totally right! 
Done